File: ObjcCrashCourse/SwiftObjCInterop/SwiftObjCInterop/ViewController.m
//
//  ViewController.m
//  SwiftObjCInterop
//
//  Created by Allen Czerwinski on 5/8/16.
//  Copyright © 2016 Allen Czerwinski. All rights reserved.
//

#import "ViewController.h"
#import "SwiftObjCInterop-Swift.h"

@interface ViewController ()
@property (nonatomic, strong) SecondVC *secondVC;
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}
- (IBAction)loadSecondVC:(id)sender {
    _secondVC = [[SecondVC alloc]init];
    _secondVC.view.backgroundColor = [UIColor blueColor];
    [self presentViewController: _secondVC animated: YES completion:nil];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Swift2iOS9Udemy/My Favorite Places - Thompson Falls/My Favorite Places - Thompson Falls/ViewController.swift
//
//  ViewController.swift
//  My Favorite Places - Thompson Falls
//
//  Created by Allen Czerwinski on 2/15/16.
//  Copyright © 2016 Allen Czerwinski. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet weak var aboutButtonUI: UIButton!
    
    @IBOutlet weak var photosButtonUI: UIButton!

    override func viewDidLoad() {
        super.viewDidLoad()
        
        aboutButtonUI.layer.cornerRadius = 4.0
        photosButtonUI.layer.cornerRadius = 4.0
        
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

